### ISSUE TITLE
Image Optimization minimumCacheTTL

### ISSUE THREAD

**Issue by cppleon** [serverless-nextjs/serverless-next.js#2192]
Currently there is no option to set `Cache-Control: max-age` header value, need to implement minimumCacheTTL
see [Caching Behavior](https://nextjs.org/docs/api-reference/next/image#caching-behavior)

**Comment by Geoff-Ford** [comment]
Yep, I have tried changing the `minimumCacheTTL` setting for `images` in the next.config.js but doesn't change the default `max-age` from 60.

**Comment by michaelroth-xx** [comment]
anything new on this one?

**Comment by devdreamsolution** [comment]
This issue is happened in NextJS 13.4.4 app router version as well.
